tests/cli-rustup.rs

```diff
@@ -1384,6 +1384,93 @@ fn file_override() {
     });
 }
 
+#[test]
+#[cfg_attr(
+    not(unix),
+    ignore = "TODO: Figure out how to write a wrapper toolchain on Windows"
```

Comment:
Would it be sufficient to install two toolchain (stable and nightly) set the default to one, and then set a `rust-toolchain.toml` to point at the other and run `rustc --version` ?